airbyte-integrations/connectors/destination-s3-glue/src/main/java/io/airbyte/integrations/destination/s3_glue/MetastoreFormatConfig.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.s3_glue;
+
+import io.airbyte.integrations.destination.s3.S3FormatConfig;
+import io.airbyte.integrations.destination.s3.util.Stringify;
+import io.airbyte.integrations.destination.s3.util.NumericType;
+
+public interface MetastoreFormatConfig extends S3FormatConfig {
+
+  String getInputFormat();
+
+  String getOutputFormat();
+
+  String getSerializationLibrary();
+
+  Stringify getStringifyType();
+
+  // TODO (quazi-h) Is this the correct place to put these methods?
+  //  This requires me to add overrides in MetastoreJsonlFormatConfig
+  NumericType getNumericType();
```

Comment:
Putting these methods here requires me to add overrides in MetastoreJsonlFormatConfig which doesn't really make sense. Should I make a separate FormatConfig for numeric types?

Reply:
For right now I think it's fine as-is. Once we add support for Avro and Parquet we may want to add another layer in between that handles these cases, but I don't think there's a much cleaner way to do this at the moment.